src/Bundle/CoverallsBundle/Collector/CiEnvVarsCollector.php

```diff
@@ -110,6 +111,48 @@ protected function fillTravisCi()
         return $this;
     }
 
+    /**
+     * Fill Github Actions environment variables.
+     *
+     * @return $this
+     */
+    protected function fillGithubActions()
+    {
+        if (!isset($this->env['GITHUB_ACTIONS'])) {
+            return $this;
+        }
+        $this->env['CI_NAME'] = 'github';
+
+        $githubEventName = $this->env['GITHUB_EVENT_NAME'];
+        $githubSha = $this->env['GITHUB_SHA'];
+        $githubRef = $this->env['GITHUB_REF'];
+
+        if (strpos($githubRef, 'refs/heads/') !== false) {
+            $githubRef = str_replace('refs/heads/', '', $githubRef);
+            $jobId = $githubSha;
+        } elseif ($githubEventName === 'pull_request') {
+            $refParts = explode('/', $githubRef);
+            $prNumber = $refParts[2];
+            $this->env['CI_PULL_REQUEST'] = $prNumber;
+            $this->readEnv['CI_PULL_REQUEST'] = $this->env['CI_PULL_REQUEST'];
+            $jobId = sprintf('%s-PR-%s', $githubSha, $prNumber);
+        } elseif (strpos($githubRef, 'refs/tags/') !== false) {
+            $githubRef = str_replace('refs/tags/', '', $githubRef);
+            $jobId = $githubRef;
+        }
+
+        $this->env['CI_JOB_ID'] = $jobId;
```

Comment:
`this->env['GITHUB_RUN_NUMBER']` is now available should be used instead of the custom commit hash based value.

https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
@OndraM maybe, but seems like maintainers ignore this PR and communication with community is broken